engine/settings/base.py

```diff
@@ -451,7 +451,7 @@
 
 GRAFANA_INCIDENT_STATIC_API_KEY = os.environ.get("GRAFANA_INCIDENT_STATIC_API_KEY", None)
 
-DATA_UPLOAD_MAX_MEMORY_SIZE = 5242880
+DATA_UPLOAD_MAX_MEMORY_SIZE = 1048576
```

Comment:
Probably worth adding a comment that it's 10MB

Reply:
Not sure, I thought it's 10MB from the first glance. So probably worth adding a comment that it's 1MB.

Reply:
Or probably make it even `1 * 1024 * 1024` to be super verbose

Reply:
we can add underscore based on PEP 515
so 1048576 will be 1_048_576